src/sentry/integrations/base.py

```diff
@@ -159,16 +160,13 @@ class IntegrationProviderSlug(StrEnum):
     IntegrationDomain.PROJECT_MANAGEMENT: [
         IntegrationProviderSlug.JIRA,
         IntegrationProviderSlug.JIRA_SERVER,
-        IntegrationProviderSlug.GITHUB,
-        IntegrationProviderSlug.GITHUB_ENTERPRISE,
-        IntegrationProviderSlug.GITLAB,
-        IntegrationProviderSlug.AZURE_DEVOPS,
```

Comment:
It's worth thinking about this a bit more. It'd be nice to have a centralized place where we can definitively state which integrations have certain meta functionality like this, but only if we can meaningfully keep it in sync with the classes themselves.

This is fine for now though.